src/txn/txn_recover.c

```diff
@@ -552,27 +553,6 @@ __wt_txn_recover(WT_SESSION_IMPL *session)
     metafile = &r.files[WT_METAFILE_ID];
     metafile->c = metac;
 
-    /*
```

Comment:
Similarly to above - why is this change here? It looks orthogonal.

Reply:
Yeah. The merge is a mess.